packages/@react-native-windows/virtualized-list/src/VirtualizedList.js

```diff
@@ -426,7 +176,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     viewOffset?: number,
     viewPosition?: number,
     ...
-  }) {
+  }): $FlowFixMe {
```

Comment:
When are these $FlowFixMe comments supposed to be addressed?

Reply:
Generally meta's process for updating the version of flow is an automated process that will make changes to the code to cast things as `$FlowFixMe` which amounts to someting similar to `any` in TS.  That way the flow update is unblocked.  Then they can go through later and provide better types.  I assume this change came from meta.